meerk40t/core/elements.py

```diff
@@ -4547,21 +4695,23 @@ def scale_elem_amount(node, scale, **kwgs):
         @self.tree_conditional(lambda node: isinstance(node.object, SVGElement))
         @self.tree_conditional_try(lambda node: not node.object.lock)
         @self.tree_submenu(_("Rotate"))
-        @self.tree_values("i", values=tuple([i for i in range(2, -14, -1) if i != 0]))
-        @self.tree_calc(
-            "angle", lambda i: "%0.f" % Angle.turns(1.0 / float(i)).as_degrees
+        @self.tree_values("angle",
+            (
+                180,150,135,120,90,60,45,30,20,15,10,9,8,7,6,5,4,3,2,1,
+                -1,-2,-3,-4,-5,-6,-7,-8,-9,-10,-15,-20,-30,-45,-60,-90,-120,-135,-150
+            )
         )
-        @self.tree_operation(_(u"Rotate turn/{i}, {angle}°"), node_type="elem", help="")
-        def rotate_elem_amount(node, i, **kwgs):
-            value = 1.0 / float(i)
+        @self.tree_operation(_(u"Rotate %s°") % ("{angle}"), node_type="elem", help="")
+        def rotate_elem_amount(node, angle, **kwgs):
+            turns = float(angle) / 360.0
             child_objects = Group()
```

Comment:
My beautiful taus!

Reply:
This was A) Horrible to use - if you wanted 38 degrees then finding a combination of rotations to get there was nigh on impossible and B) broken anyway - the list was a fraction of the size it should have been.